### PR TITLE
Fixes Settings page error

### PR DESCRIPTION
**Pull request by Remorax**
Fixes issue #370 

Changes:
The settings page bug was fixed, by taking light as the default theme when user logs in for the first time. This can be viewed by taking a clone of my fork and running it on localhost.

Please review this ASAP :)

### CODE REVIEW

File: src/components/Settings/Settings.js [hunk 1]
```diff
@@ -59,8 +59,8 @@ class Settings extends React.Component {
         }
     }
     componentDidMount(){
-        let state = this.state;
-        let url = BASE_URL+'/aaa/listUserSettings.json?'
+        var state = this.state;
```
Comment (anshumanv) [inline src/components/Settings/Settings.js]: Any particular reason for this change?

**Comment by anshumanv** [comment]
Please provide a test link if possible.

**Comment by Remorax** [comment]
I can provide a test link and it works there but since the bug was on localhost, it would be better if you tested it there only.
Anyway here's the demo link: silent-chance.surge.sh

File: src/components/Settings/Settings.js (hunk 1 above)
Comment (Remorax) [reply src/components/Settings/Settings.js]: No, I was just trying out various changes and I forgot to revert this

Comment (anshumanv) [reply src/components/Settings/Settings.js]: No issues, good work. :smile:

**Comment by anshumanv** [comment]
Ah, yes @Remorax the error is not reproducible on the test link.

**Comment by Remorax** [comment]
@madhavrathi @isuruAb @rishiraj824 @Orbiter Please merge this ASAP :)

**Comment by isuruAb** [comment]
@saurabhjn76  please review this

**Pull request merged**